### DISCUSSION TITLE
SABnzbd - Update script fails

### DISCUSSION THREAD

**Discussion by maps119** [tteck/Proxmox#3349] [Help Needed]
Hi,

Just trying to update my SABnzbd LXC to the latest version. It was deployed originally via tteck script.

Tried running "update" as well as the wget but getting the below error.

 / Updating SABnzbd   
[ERROR] in line 65: exit code 0: while executing command python3 -m pip install -r requirements.txt &> /dev/null

Let me know if you need any additional information.

Thanks in advanced!

**Comment by maps119** [comment]
Just to update, I deployed a replacement LXC and restored the backup from SAB so not really an issue for me anymore but may help others in the future if anything needs to be adjusted in relation to the error.

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/discussions/3300
